lib/src/mount.rs

```diff
@@ -57,3 +57,19 @@ pub(crate) fn mount(dev: &str, target: &Utf8Path) -> Result<()> {
         [dev, target.as_str()],
     )
 }
+
+// Ensure the `/dev` directory exists.
+fn ensure_mountpoint(target: &str) -> Result<()> {
+    std::fs::create_dir_all(target)?;
+    Ok(())
+}
+
+/// General mount that also checks that mount path exists
+pub(crate) fn ensure_mount(dev: &str, target: &str, fstype: &str) -> Result<()> {
+    ensure_mountpoint(target)?;
```

Comment:
I had actually expected this call to be handling *idempotence* - i.e. we don't do the mount if the target is already mounted with the expected type.

The current docs gained a `-v /dev:/dev` recommendation for this reason, and I think we don't want to overmount it.